pgmpy/models/MarkovChain.py

```diff
@@ -363,3 +363,37 @@ def random_state(self):
         [State('diff', 2), State('intel', 1)]
         """
         return [State(var, np.random.randint(self.cardinalities[var])) for var in self.variables]
+
+    def copy(self):
+        """
+        Returns copy of Markov Chain
+
+        Return Type:
```

Comment:
@anchitjain1234 Please follow the docstring format exactly. Check docstrings for other methods.